.travis.yml

```diff
@@ -1,83 +1,96 @@
-      osx_image: xcode10.2 # macOS 10.14
-      env:
-        - TASK="clang, ROOT v6.18"
-        - ROOTBIN="v6.18.04.macosx64-10.14-clang100"
-        # Add the brew python binary path to ensure python3 will be used
-        - PATH=/usr/local/opt/python/libexec/bin:$PATH
-      before_install:
-        - echo "Using system python version."
 
-    - os: linux
-      dist: bionic # gcc 7.4
-      compiler: gcc
-      env: 
-        - TASK="gcc7, ROOT v6.18"
-        - ROOTBIN="v6.18.04.Linux-ubuntu18-x86_64-gcc7.4"
-      before_install:
-        # TravisCI uses pyenv to manage different python versions
-        # FindPython3.cmake finds always this version (no matter 
-        # if it is activated or not)
-        - pyenv global 3.7.1
-        - pip3 install pipenv
+  - os: osx
+    osx_image: xcode10.2
+    addons:
+      homebrew:
+        packages:
+        - boost
+        - gsl
+        - python
+        - pipenv
+    env:
+    - TASK="clang, ROOT v6.18"
+    - ROOTBIN="v6.18.04.macosx64-10.14-clang100"
+    - PATH=/Users/travis/Library/Python/3.7/bin:$PATH
+    before_install:
+    - echo "Using system python version."
+
+  - os: linux
+    dist: bionic
+    compiler: gcc
+    env:
+    - TASK="gcc7, ROOT v6.18"
+    - ROOTBIN="v6.18.04.Linux-ubuntu18-x86_64-gcc7.4"
+    addons:
+      apt:
+        update: true
+        packages:
+        - libboost-all-dev
+        - libgsl0-dev
+    before_install:
+    - pyenv global 3.7.1
+    - pip3 install pipenv
+
+  - os: linux
+    dist: bionic
+    env:
+    - TASK="deployment"
+    before_install:
+    - pyenv global 3.7.1
+    install: echo "Skipping install stage!"
+    script: echo "Skipping script stage!"
+    before_deploy:
+    - pip3 install scikit-build
+    deploy:
+      - provider: pypi # Upload new release on pypi for each tagged commit
+        on:            # Make sure that the version number in setup.py is 
+          tags: true   # increased before(!) you tag the commit
+        distributions: sdist
+        skip_existing: true
+        user: __token__
+        password:
+          secure: bvlu0vG8oOYgLbNRkbZT5QO8AQDRlMuHPUn4U7PIECzQNjFC31Y8ugXTQOPdcNsACc8TR3z52uZ1JmuK4XI9++7HuzRwhMgKziCEwD/fBdZETxXq95LorbMaD8qOY43v5vJAeFDzouDoP3iBVTpJG3dZPk7OrHy0uSSGYnC0M/WqoFg/3Lfg5tlKz8doBbgGVG/vY/58GyDxu3GcSVg7c3A4CNYKk6FwXwbWo7itTzXMJFjWmlolFtNv9m4b4gLrPSvUf7A/BrovC9IYic2N+T/KnbVOBopQ9ggk+uf1eMv1emruNiqUaJ963DMPjE154hbR7Ykvz4ng/CsvI8enmiPg18RoS1DLGxe7LVr0RzZaGsW0gw8q6H6Noc0FzNXwLb5+fDWHJtz87rrVmZZz+ofVpo370Q9y7B6joJ5EBDxPwi/Ef9MBZMEMR1pArX9rnlkVpPit0nvsb+R4e5ZS5CHj7ArSxio5pXLMrEtJyJKpnaeCbjIp5b+UfsVwSbY6i3UpE0HSHDW/Nic02/d9PfgxuBkEp6jb30yajsfWroMpJQyKD/X1qIUO48deUO79Mnu33EGWnccYBbmhMxFNCQnfLVPwKAQm7YM1/WlUa6ictifxlLouG8wZjAUviBcvTjzDY53UUpS/PIkGPL9MrqjLJbsR0XwAwp4leprngyY=
+      # - provider: release # Create github release on each tagged commit
+      #   api_key:
+      #   file: "FILE TO UPLOAD"
+      #   skip_cleanup: true
+      #   draft: true
+      #   on:
+      #     tags: true
```

Comment:
I would avoid uploading commented out code

Reply:
Took me a while to find those lines so I would like to keep it